### ISSUE TITLE
Strange records in EXEC_CONTEXT_PIPELINE and other DB issues

### ISSUE THREAD

**Issue by tomas-knap** [mff-uk/odcs#149]
When looking into the table  EXEC_CONTEXT_PIPELINE, I can see:

![screen shot 2013-07-18 at 4 07 53 pm](https://f.cloud.github.com/assets/3014917/819535/939648ce-efb3-11e2-8e21-38007f44df20.png)

If I understand it correctly, every record in EXEC_CONTEXT_DPU points to the EXEC_CONTEXT_PIPELINE, so if I have two dpus on the pipeline, there will be two records in the EXEC_CONTEXT_DPU pointing to one record in EXEC_CONTEXT_PIPELINE. Am I correct?

So why the redundancy in EXEC_CONTEXT_PIPELINE?

**Comment by tomas-knap** [comment]
Also my question is, what should the table EXEC_CONTEXT_PROCCONTEXT include?

**Comment by tomas-knap** [comment]
Execution_directory in EXEC PIPELINE, what is the purpose of this column?

**Comment by janvojt** [comment]
Table `EXEC_CONTEXT_PIPELINE` represents entity `ExecutionContextInfo`. The problem is, that there are multiple instances of this object for the same execution created in backend. This is wrong, I will research this further and maybe discuss it with @skodape.

Table `EXEC_CONTEXT_PROCCONTEXT` is not in schema.sql. Maybe just a leftover table from previous revision?

Column `execution_directory` in table `EXEC_PIPELINE` is not used. It was added by @skodape in merge commit, probably by a mistake. I removed it.
